### PR TITLE
Journalist can add images to articles

### PR DESCRIPTION
**Pull request by EevanR**
[PT LINK](https://www.pivotaltracker.com/story/show/170646674)

- install active storage
- add amazon storage
- association between article and image
- image spec tests when journalists create
- update articles controller

### CODE REVIEW

File: app/services/decode_service.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module DecodeService
+	def self.attach_image(file, target)
+		image = Rails.env.test? ? file : split_base64(file)
+
```
Comment (oliverochman) [inline app/services/decode_service.rb:4]: indentation

File: config/storage.yml [hunk 2]
```diff
@@ -7,12 +7,12 @@ local:
...
-#   access_key_id: <%= Rails.application.credentials.dig(:aws, :access_key_id) %>
-#   secret_access_key: <%= Rails.application.credentials.dig(:aws, :secret_access_key) %>
-#   region: us-east-1
-#   bucket: your_own_bucket
+amazon:
+  service: S3
+  access_key_id: <%= Rails.application.credentials.dig(:aws, :access_key_id) %>
+  secret_access_key: <%= Rails.application.credentials.dig(:aws, :secret_access_key) %>
+  region: us-east-1
+  bucket: your_own_bucket
```
Comment (oliverochman) [inline config/storage.yml]: you need to ask thomas for a bucket

File: spec/requests/api/admin/create_spec.rb [hunk 3]
```diff
@@ -18,6 +28,11 @@
     it 'returns a 200 response status' do
       expect(response).to have_http_status 200
     end
+
+    it 'returns an attached image' do
```
Comment (oliverochman) [inline spec/requests/api/admin/create_spec.rb]: why would it return an image?

**Review by oliverochman — DISMISSED** [review]

File: spec/requests/api/admin/create_spec.rb (hunk 3 above)
Comment (EevanR) [reply spec/requests/api/admin/create_spec.rb]: ah yeah. Thats poor wording there, has been changed.

File: app/services/decode_service.rb (hunk 1 above)
Comment (EevanR) [reply app/services/decode_service.rb:4]: yet again, github has changed the indentation to 8, indenting is normal and set to 2 on my vscode...

Comment (EevanR) [reply app/services/decode_service.rb:4]: not sure why this keeps happening

File: config/storage.yml (hunk 2 above)
Comment (EevanR) [reply config/storage.yml]: got it

**Review by tochman — APPROVED** [review]
Looks good

**Pull request merged**